examples/pytorch/MNIST/example2/README.md

```diff
@@ -0,0 +1,78 @@
+## MNIST example with MLFlow
+
+In this example, we train a model to predict handwritten digits.The autolog code uses Pytorch Lightning's MLFlowLogger to log metrics. 
+The code is trained using pytorch lightning loop and the autolog function call in the main - `autolog()`
+is responsible for logging the params, metrics, model summary and the model.
+This example logs metrics only after n epoch iterations. The iteration limit can be set in the autolog method using the parameter `log_every_n_iter=NUMBER-OF-ITERATIONS`.
+For ex: `autolog(log_every_n_iter=5)`
```

Comment:
Could we remove this in favor of linking to the ``mlflow.pytorch.autolog`` doc once it's available, as described in https://github.com/mlflow/mlflow/pull/3608/files#r514601992?